### PR TITLE
mgr/telemetry: don't call obsolete function, English nit

### PR DESCRIPTION
**Pull request by dmick**
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by tchaikov** [comment]
i think #26921 should address the `set_config()` issue.

File: src/pybind/mgr/telemetry/module.py [hunk 1]
```diff
@@ -272,15 +272,15 @@ def handle_command(self, inbuf, command):
                 r[opt['name']] = getattr(self, opt['name'])
             return 0, json.dumps(r, indent=4), ''
         elif command['prefix'] == 'telemetry on':
-            self.set_config('active', True)
+            self.set_module_option('active', True)
```
Comment (votdev) [inline src/pybind/mgr/telemetry/module.py]: s/active/enabled/

**Comment by votdev** [comment]
Thanks for the contribution, but @liewegas is already working on that.
Duplicate of #26921.

File: src/pybind/mgr/telemetry/module.py (hunk 1 above)
Comment (dmick) [reply src/pybind/mgr/telemetry/module.py]: doh, yes

**Pull request closed without merging**